### PR TITLE
WIP: unify param def

### PR DESCRIPTION
**Pull request by 0xMuluh**
This PR aims to harmonize the miaViz doc.

- use template: param --> type --> def --> Default

### CODE REVIEW

File: R/plotAbundanceDensity.R [hunk 1]
```diff
@@ -51,31 +40,30 @@
...
-#'   \item{point_size}{ a positive numeric value selecting the size of point in
-#'   \code{jitter} and \code{point} plot. (default: \code{point_size = 2}) }
+#'   \item \code{point_size} \code{Positive integer scalar}. Selects the size of point in
+#'   \code{jitter} and \code{point} plot. (Default: \code{2})
 #'   
-#'   \item{add_legend}{ a boolean value selecting if legend is added. 
-#'   (default: \code{add_legend = TRUE}) }
+#'   \item \code{add_legend} \code{Logical scalar}. Determines if legend is added. 
+#'   (Default: \code{TRUE})
 #'
```
Comment (TuomasBorman) [inline R/plotAbundanceDensity.R]: These should be point.shape (parameter.name)?

**Review by TuomasBorman — CHANGES_REQUESTED** [review]
Looks good, see comment

File: R/plotAbundanceDensity.R (hunk 1 above)
Comment (0xMuluh) [reply R/plotAbundanceDensity.R]: Thanks.

File: R/plotAbundanceDensity.R [hunk 2]
```diff
@@ -51,31 +40,30 @@
...
+#'   (Default: \code{"Taxa"})
 #'   
-#'   \item{point_alpha}{ a numeric value from range 0 to 1 selecting the transparency of
-#'   colour in \code{jitter} and \code{point} plot. (default: \code{point_alpha = 0.6}) }
+#'   \item \code{point.alpha} \code{Numeric scalar}. From range 0 to 1. Selects the transparency of
+#'   colour in \code{jitter} and \code{point} plot. (Default: \code{0.6})
 #'   
-#'   \item{point_shape}{ a positive integer value selecting the shape of point in
-#'   \code{jitter} and \code{point} plot. (default: \code{point_shape = 21}) }
+#'   \item \code{point.shape} \code{Positive integer scalar}. Value selecting the shape of point in
```
Comment (TuomasBorman) [inline R/plotAbundanceDensity.R:53]: In the example below, there is point_shape. Can you change it to point.shape?

File: R/plotGraph.R [hunk 3]
```diff
@@ -10,74 +10,72 @@
...
 #'   
-#' @param name If \code{x} is a 
+#' @param name \code{Character scalar}. If \code{x} is a 
 #' \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}
-#' the key for subsetting the \code{metadata(x)} to a graph object.
+#' the key for subsetting the \code{metadata(x)} to a graph object. (Default: \code{"graph"})
 #' 
-#' @param show.label \code{logical} (scalar), \code{integer} or \code{character}
-#'   vector. If a \code{logical} scalar is given, should tip labels be plotted
+#' @param show.label \code{Logical scalar}, \code{integer scalar} or \code{character vector}
```
Comment (TuomasBorman) [inline R/plotGraph.R]: integer vector

File: R/plotPrevalence.R [hunk 4]
```diff
@@ -34,59 +34,53 @@
 #' @param assay.type a \code{character} value defining which assay data to
```
Comment (TuomasBorman) [inline R/plotPrevalence.R]: character scalar

**Review by TuomasBorman — APPROVED** [review]

**Pull request merged**